src/main/java/dune/DateTimePrinter.java

```diff
@@ -0,0 +1,37 @@
+package dune;
+
+import java.time.LocalDateTime;
+
+/**
+ * Deals with formatting the date and time.
+ */
+public class DateTimePrinter {
+
+    /** Months in the year. */
+    private String[] months = new String[] {
```

Comment:
Perhaps use a more detailed name here (e.g. = listOfMonths, nameOfMonths)? Because it might cause some misleading like thinking it of a variable used to store months used.